src/resources/shipment.js

```diff
@@ -169,6 +169,31 @@ export default (api) =>
       return this.rpc('insure', { amount });
     }
 
+    /**
+     * Generate a form for a shipment.
+     * @param {string} formType
+     * @param formOptions
+     * @returns {this}
+     */
+    async generateForm(formType, formOptions) {
+      // eslint-disable-next-line no-param-reassign
+      formOptions = formOptions || {};
```

Comment:
Issue: Variable reassignment can be dangerous which is why the linter is angry it's happening. Let's not ignore it but follow the linter - it's here to help.

Reply:
Resolved